addons/website/static/src/snippets/s_product_catalog/options.js

```diff
@@ -16,19 +16,15 @@ options.registry.ProductCatalog = options.Class.extend({
      */
     toggleDescription: function (previewMode, widgetValue, params) {
         const $dishes = this.$('.s_product_catalog_dish');
-        const $name = $dishes.find('.s_product_catalog_dish_name');
-        $name.toggleClass('s_product_catalog_dish_dot_leaders', !widgetValue);
         if (widgetValue) {
             $dishes.toArray().forEach((el) => {
                 const $description = $(el).find('.s_product_catalog_dish_description');
                 if ($description.length) {
                     $description.removeClass('d-none');
```

Comment:
Ok, I reintroduced these lines, I let you handle it in your task 👍

Reply:
I'm sorry @Brieuc-brd , I wasn't clear here, I was talking about line 25.
I left this comment because I was wondering what this code was for since we never use it, and I wanted to share my findings.
Lines 19 and 20 can be removed as you did.